src/main/java/uk/gov/pay/api/agreement/resource/AgreementsApiResource.java

```diff
@@ -83,7 +83,7 @@ public SearchResults<Agreement> getAgreements(@Auth Account account, @BeanParam
 
     @POST
     @Path("/v1/agreements/{agreementId}/cancel")
-    @Consumes("application/json")
+    @Produces(APPLICATION_JSON)
```

Comment:
Is `204 NO CONTENT` strictly `application/json`? I'm not actually sure

Reply:
This is what was causing me grief earlier. Normally there’s no content but when there’s an error there’s a JSON body so we need this or Jersey will panic and we end up with a 500.